app/core/utils.js

```diff
@@ -11,6 +13,43 @@ function getAddressArrayFromBuilding(building) {
   return '';
 }
 
+function getAddressString(address) {
+  const street = address.street_name_fi || '';
+  const municipality = address.municipality_fi || '';
+  return `${street}\, ${municipality}`;
+}
```

Comment:
I have created another function inside SchoolStore that grabs the address from a building. Perhaps we could make that function to use this getAddressString function as well .